manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -80,6 +80,16 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: liveness-probe
+          image: quay.io/k8scsi/livenessprobe:v1.1.0
+          args:
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
```

Comment:
Is env required here? we can directly use it?

Reply:
please refer to all above usage (other containers)
they use ADDRESS so I want to be consistent